source/fuzz/fact_manager.cpp

```diff
@@ -1401,6 +1401,9 @@ class FactManager::IrrelevantValueFacts {
   // See method in FactManager which delegates to this method.
   bool IdIsIrrelevant(uint32_t pointer_id) const;
 
+  // See method in FactManager which delegates to this method.
+  std::unordered_set<uint32_t> GetIrrelevantIds() const;
```

Comment:
I think it's better to make it a const lvalue reference (`const std::unordered_set<uint32_t>&`).

Reply:
Thanks!